config/locales/en.yml

```diff
@@ -1468,6 +1466,13 @@ en:
           referenced_in_other_fields_html:
             one: "%{name} is used in project attribute calculation %{links}."
             other: "%{name} is used in project attribute calculations: %{links}."
+          attributes:
+            formula:
+              blank: "Formula can't be blank."
+              invalid: "Formula is invalid."
+              invalid_characters: "Only numeric values, project factors, calculated values and mathematical operators are allowed."
```

Comment:
By now the lists have been (or will be) renamed to "Weighted item list" and the message should reflect this.

Maybe: "Only numeric values, mathematical operators and project attributes of type integer, float, calculated value and weighted list are allowed.